packages/connect-react/src/index.ts

```diff
@@ -184,52 +211,108 @@ export function usePermissions(): [Permission[], LoadingStatus] {
   return useConnectSubscription<Permission[]>(callback, [])
 }
 
-export function createAppHook(appConnect: Function) {
+export function createAppHook(
+  appConnect: Function,
+  connector?: string | [string, { [key: string]: any } | undefined]
```

Comment:
I'm not familiar with the type set-up here but should this connector prop expect `ConnectorDeclaration`?

Reply:
Almost! The exception is that we can’t pass a [`IOrganizationConnector`](https://github.com/aragon/connect/blob/6e16015925133a9649e2c6533f39ead3303789a6/packages/connect/src/connect.ts#L30) here, as it is only meant for the main (organization) connector, not the app connectors that only have a name + a configuration.